sources/src/ChatGPT.tsx

```diff
@@ -449,7 +460,7 @@ function ChatGPT(props: ChatGPTProps) {
           disabled={streaming}
           label="Send a message"
           value={prompt}
-          onChange={(e) => setPrompt(e.target.value)}
+          onChange={(e) => handleInputChange(e.target.value)}
```

Comment:
This can probably be rolled back based on the message above regarding setting hadConversation

Reply:
Updated